INSTALLATION.rst

```diff
@@ -48,55 +48,66 @@ Recommended Method
 
 .. attention::
 
-   There are many ways (described further down) to obtain the software
+   There are many ways to obtain the software
    packages necessary to run :term:`FiPy`, but the most expedient way is
-   with the conda_ package manager.
+   with the conda_ package manager.  In addition to the scientific
```

Comment:
As a proper noun, Conda ought to be capitalized except when referring to `conda` explicitly.

Reply:
I squander no neurons trying to figure out how to capitalize things when they can't figure it out themselves.

Reply:
AKA, find me a case where they call themselves "Conda" (and not "CONDA" or "conda") that's not at the beginning of a sentence and I might begin to care

Reply:
OK -- FiPy style is to take the lead of the entity referred to by the proper noun, not to blindly capitalize proper nouns as most style guides (including The Carpentries) would have us do.

Reply:
FiPy style is to be respectful of what entities choose to call themselves unless they make it a PITA to figure out what that is